src/telemetry/index.ts

```diff
@@ -1,3 +1,4 @@
 import TelemetryService from './telemetryService';
+export * from './telemetryEvents';
 
-export { TelemetryService };
+export default TelemetryService;
```

Comment:
this is a more controversial preference thing but can we consider not using default exports (and maybe even banning them eventually...)? I know TelemetryService was actually mainly being used as a default export and we have a lot in the codebase.

I think benefits of it are quite minor though, nothing that life changing.

https://www.lloydatkinson.net/posts/2022/default-exports-in-javascript-modules-are-terrible/
https://cichocinski.dev/blog/using-default-exports-makes-javascript-harder

Reply:
I'd not be opposed to that - the only reason I reexported it as default was to avoid changing a bunch of places where we were importing it from `telemetry/telemetryService`.

Reply:
Yeah that makes sense, it may be worth leaving for a different time and dealing with default exports more globally later